tests/tasks/teardown/awscli-eks.yaml

```diff
@@ -37,6 +39,13 @@ spec:
           aws eks wait nodegroup-deleted --nodegroup-name $i --cluster-name $(params.cluster-name) $ENDPOINT_FLAG --region $(params.region);
       done;
       aws eks delete-cluster --name $(params.cluster-name) --region $(params.region) $ENDPOINT_FLAG
+  - name: teardown-eks-role-stack
+    image: alpine/k8s:1.23.13
+    script: |
+      echo "Approving KCM requests"
+      kubectl certificate approve $(kubectl get csr | grep "Pending" | awk '{print $1}')  2>/dev/null || true
```

Comment:
I don't see a need to approve CSRs.
There are lightly to have tens of thousands of CSR accumulated over the time that the test are run. Each kubectl will create multiple. It can take a long time to approve all of them.

Reply:
yeah makes sense.. Have removed it.